src/compiler/compile/render_dom/wrappers/Element/index.ts

```diff
@@ -155,6 +159,26 @@ export default class ElementWrapper extends Wrapper {
 		next_sibling: Wrapper
 	) {
 		super(renderer, block, parent, node);
+
+		if (node.is_dynamic_element) {
```

Comment:
```suggestion
		if (node.is_dynamic_element && block.type !== 'child_dynamic_element') {
```

Reply:
Thank you! I updated it on this commit!
https://github.com/sveltejs/svelte/pull/6898/commits/b437985e0dadfc6eb635289c26ba44d495091ee4